### DISCUSSION TITLE
Custom svg generation for reducing file size

### DISCUSSION THREAD

**Discussion by nathan-yuen** [szimek/signature_pad#633] [Ideas]
Is there any chance of supporting custom svg generation?

Right now every `<path>` has its own `stroke`, `fill`, & `stroke-linecap`.
All these attributes could be replaced with a scoped `<style>` if I only need to support single stroke color?
And it also help reduce file size significantly!

Exmaple
```xml
<style scoped>
  path {
    stroke-linecap: round;
    stroke: #000;
    fill: none;
  }
</style>
```

**Comment by UziTech** [comment]
PRs are welcome 😁👍

**Comment by UziTech** [comment]
There are already libraries for optimizing svgs like [svgo]( https://www.npmjs.com/package/svgo).

**Reply by UziTech** [reply]
Ya doing replaces will work but it isn't worth putting it in this library when there are libraries that can do it much better.
